test/bin/minimal-update.t/run.t

```diff
@@ -0,0 +1,68 @@
+We have a simple project with a single package which depends on
+packages a and b: 
```

Comment:
Just a small thing: it would be nice if these were `a`, `b`, `c` in the readme, because sometimes it is hard to distinguish between the package `a` and the word "a".